src/page/DuesManagement/index.tsx

```diff
@@ -0,0 +1,235 @@
+import { Link, useLocation } from 'react-router-dom';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Pagination from '@mui/material/Pagination';
+import PaginationItem from '@mui/material/PaginationItem';
+import { Button, Checkbox, FormControl, FormControlLabel, FormGroup, FormLabel, Input, Popover } from '@mui/material';
+import Modal from "@mui/material/Modal";
+import { ChangeEvent, useState } from 'react';
+import { useGetAllDues } from 'query/dues.ts';
+import useBooleanState from 'util/hooks/useBooleanState.ts';
+import { DuesDetail } from 'model/dues/allDues.ts';
+import { ALL_TRACKS } from 'util/constants/alltracks.ts';
+import * as S from './style.ts';
+
+export default function DuesManagement() {
+  const location = useLocation();
+  const query = new URLSearchParams(location.search);
+  const page = parseInt(query.get('page') || '1', 10);
+  const thisYear = new Date().getFullYear();
+  const duesYear = thisYear - page + 1;
+  const [track, setTrack] = useState([true, true, true, true, true, true]);
+  const [name, setName] = useState("");
+  const [detail, setDetail] = useState<DuesDetail>({ month: 0, status: "" });
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const memoPopOverOpen = Boolean(anchorEl);
+  const { value: isFilterModalOpen, setTrue: openFilterModal, setFalse: closeFilterModal } = useBooleanState(false);
+
+  const { allDues } = useGetAllDues(duesYear);
```

Comment:
suspense Query를 썻다면 해당 부분을 `<Suspense /> 로 감싸줄 필요가 있겠죠?
지금 Route를 전부 감싼 친구는 #9 에서 지워지는 것 같아요

Reply:
이 부분에서 궁금한 점이 있어서 PR에 내용을 남기려고 했는데 깜빡했네요.
`allDues`를 사용하는 부분에 `Suspense`를 사용하여 `LoadingSpinner`를 적용하려고 했는데, 다음과 같은 에러가 발생하는데, 어떻게 해결해야 할 지 모르겠습니다...
```
Uncaught Error: A component suspended while responding to synchronous input. 
This will cause the UI to be replaced with a loading indicator. 
To fix, updates that suspend should be wrapped with startTransition
```

Reply:
1. 쓰는 곳을 분리해 Suspense로 묶고, 내부에서 쿼리를 호출한다
```ts
<section>
  <Suspense>
    <TableUI />
  </Suspense>
</section>
```

2. (보통은 컴포넌트 전체에서 쿼리를 필요로 하는경우) 해당 컴포넌트를 묶어서 Export한다
```ts
// before
export function TablePage(){}

// after
function Table() {}

export function TablePage(){
  return (
    <Suspense>
       <Table />
    </Suspense>
  )
}

```

Reply:
2번째 방법을 사용해서 내용 수정했습니다! bcaf1b07ff77dcfa336d36f3b20cb991a858f677